integration/cri-o/cri-o.sh

```diff
@@ -69,29 +69,35 @@ IFS=''
 
 # Skip CRI-O tests that currently are not working
 pushd "${crio_repository_path}/test/"
-cp ctr.bats ctr_kata_integration_tests.bats
-for testName in "${!skipCRIOTests[@]}"
-do
-	sed -i '/'${testName}'/a skip \"'${skipCRIOTests[$testName]}'\"' "ctr_kata_integration_tests.bats"
-done
 
-# selectively skip tests depending on the version of cri-o we're testing with
 CRIO_VERSION=$(echo ${PULL_BASE_REF} | cut -d'-' -f 2)
-echo GOT CRIO VERSION $CRIO_VERSION
-if [ "$CRIO_VERSION" != "main" ]; then
-    if [ -z "$CRIO_VERSION" ]; then
-        echo "Unknown version of cri-o - skipping more tests"
-        CRIO_VERSION="1.21"
-    fi
+if [ -z "$CRIO_VERSION" ]; then
+    echo "Unknown version of cri-o - skipping more tests"
+    CRIO_VERSION="1.21"
+else
+    echo GOT CRIO VERSION $CRIO_VERSION
+fi
 
-    for testName in "${!fixedInCrioVersion[@]}"
+for batsfile in ${bats_files_list[@]}; do
+    testfile=${batsfile%.*}_kata_integration_tests.bats
+    cp ${batsfile} ${testfile}
+    for testName in "${!skipCRIOTests[@]}"
```

Comment:
We are preparing for multiple bats file here (yay)!
The code here will try to skip all the skipCRIOTests from each bat file.
While this will work (as if the test is not found nothing bad will happen) maybe we can have separate skipCRIOTests arrays for each bats file (e.g., ctr_skipCRIOTests) and just skip the test for each bats file (so also in crio_skip_tests.sh we will have tests to skip better grouped there, i.e., in different array vars).
We can do that with something  like:
`eval skip_list=\$\{\!${batsfile%.*}_skipCRIOTests\[\@\]\}`
`for testName in $skip_list`

Reply:
Thanks @fgiudici 
I ended up using an alias (declare -n skip_list=${batsfile}_skipCRIOTests) and using "skip_list" everywhere after.
It required less \ characters :-)
I'm also doing it for the "fixedInCrioVersion" list.
I'll send the commit after testing.